README.md

```diff
@@ -1,6 +1,8 @@
 # CKEditor 4 - The best browser-based WYSIWYG editor
 
 [![devDependencies Status](https://david-dm.org/ckeditor/ckeditor-dev/dev-status.svg)](https://david-dm.org/ckeditor/ckeditor-dev?type=dev)
+[![Build Status](https://travis-ci.org/ckeditor/ckeditor-dev.svg?branch=major)](https://travis-ci.org/ckeditor/ckeditor-dev)
```

Comment:
Actually it's hardcoded to `major` branch. Can you adjust so that it picks master/major branch properly? It can't be hardcoded as it will cause problems during merges.